README.md

```diff
@@ -10,7 +20,9 @@ Wave Recovery Tool is licensed under the terms of the GNU General Public License
 
 ## Author
 
-Wave Recovery Tool is developed by David Pace &lt;dev@davidpace.de&gt;
```

Comment:
You removed this line, please add it again. Instead, remove the `## Author` heading since we already have the `## Authors` heading.